src/pages/rekey-reason/rekey-reason.ts

```diff
@@ -303,4 +337,23 @@ export class RekeyReasonPage extends BasePageComponent {
     }
   }
 
+  checkUserExists() {
+    this.checkingStaffExists = true;
```

Comment:
This should be triggered via actions and an effect instead of directly calling the provider from the page